docs/environments.adoc

```diff
@@ -20,9 +20,9 @@ struct custom_receiver {
   friend auto tag_invoke(async::set_stopped_t) { ... }
 
   struct env {
-    [[nodiscard]] friend constexpr auto tag_invoke(async::get_stop_token_t,
-                                                   env const& e) -> async::inplace_stop_token {
-      return e.token;
+    [[nodiscard]] constexpr auto tag_invoke(async::get_stop_token_t) const
```

Comment:
question: shouldn't this be using query?

Reply:
Yes! good spot.